# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1319035589 -3600
#      Wed Oct 19 15:46:29 2011 +0100
# Node ID 6d2127468d96f4dec5e3d3ea86fed4dbdc241f10
# Parent  c943be5f929da853b3705283d0aa08bdde227734
Revert changeset c943be5f929d as it breaks Linux compilation.

* arpack/util/second.f (ETIME): Revert declaration back to INTRINSIC.

diff --git a/libcruft/arpack/util/second.f b/libcruft/arpack/util/second.f
--- a/libcruft/arpack/util/second.f
+++ b/libcruft/arpack/util/second.f
@@ -16,17 +16,17 @@
 *     .. Local Scalars ..
       REAL               T1
 *     ..
 *     .. Local Arrays ..
       REAL               TARRAY( 2 )
 *     ..
 *     .. External Functions ..
       REAL               ETIME
-      EXTERNAL           ETIME
+      INTRINSIC          ETIME
 *     ..
 *     .. Executable Statements ..
 *
 
       T1 = ETIME( TARRAY )
       T  = TARRAY( 1 )
 
       RETURN
